packet/ldap.py

```diff
@@ -1,14 +1,14 @@
 from functools import lru_cache
 
-import ldap
-
 from packet import _ldap
 
 
+@lru_cache(maxsize=4096)
```

Comment:
4096 seems a bit large for groups, no? We only have a handful of groups at maximum.